File: Python Word examples/tables/update_table_cell_format.py
import spirecloudword
from spirecloudword.api.tables_api import TablesApi
from spirecloudword.models.table_cell_format import TableCellFormat
from spirecloudword.models.color import Color
from spirecloudword.configuration import Configuration as WordConfiguration

appId = "your id"
appKey = "your key"

configuration = WordConfiguration(appId, appKey)
api = spirecloudword.api.tables_api.TablesApi(configuration)

filename = "Template.docx"
index = 0
table_row_path = "sections/0/tables/0/rows/0"
cell_format = TableCellFormat()
cell_format._background_color = Color(100, 100, 100)
cell_format._width = 80
inputFolder = "input"
outputFilePath = "output/Tables/update_table_cell_format.docx"
result = api.update_table_cell_format(filename, table_row_path, index, cell_format, folder=inputFolder, dest_file_path=outputFilePath)

File: JavaScript Word Examples/TablesApiDemo/Cells/updateTableCellFormat.js
(function (){
    var appId = "your id";
    var appKey = "your key";
    var baseUrl = "https://api.e-iceblue.cn";

    var Spirecloudword = require('../../../src/index');
    var apiClient = new Spirecloudword.ApiClient(appId, appKey,baseUrl);
    var instance = new Spirecloudword.TablesApi(apiClient); 

    var filename = "Table.docx";
    var index = 5;
    var format = new Spirecloudword.TableCellFormat();
    format.width = 100;
    format.backgroundColor = new Spirecloudword.Color();
    format.backgroundColor.red = 100;
    format.backgroundColor.green = 100;
    format.backgroundColor.blue = 100;
    var tableRowPath = "Section/0/Body/0/Table/0/TableRow/3";
    var desFilePath = "output/Tables/UpdateTableCellFormat.docx";
    var opts = { folder: 'Tables' }
    instance.updateTableCellFormat(filename, tableRowPath, index, format, desFilePath, opts,
    function (error, data, response) {
         if (error) throw error;
    });
})();